src/main/java/com/github/tomakehurst/wiremock/client/HttpAdminClient.java

```diff
@@ -165,8 +164,7 @@ private String postJsonAssertOkAndReturnBody(String url, String json, int expect
         HttpPost post = new HttpPost(url);
         try {
             if (json != null) {
-                StringEntity stringEntity = new StringEntity(json);
-                stringEntity.setContentType(APPLICATION_JSON.getMimeType());
+                StringEntity stringEntity = new StringEntity(json, APPLICATION_JSON);
```

Comment:
Nice catch. I didn't know that, but from the implementation point of view (in StringEntity) it is easier.

Btw, @mariuszs maybe you should report it upstream (as an possible engancement)?